src/store.tsx

```diff
@@ -12,6 +12,7 @@ export type RedisCredentials = {
 
 type DatabrowserContextProps = {
   redis: Redis
+  redisNoPipeline: Redis
```

Comment:
in the end, what do we use redis for, and what do we use redisNoPipeline for?

Can you write these to PR description

Reply:
Added